tests/scale_test/utils.py

```diff
@@ -0,0 +1,238 @@
+import json
+import uuid
+import requests
+import click
+
+
+def send_notification(lira_url, auth_token, notification):
+    """Send a notification to a given Lira.
+
+    Args:
+        lira_url (str): A typical Lira url, e.g. https://pipelines.dev.data.humancellatlas.org/
+        auth_token (str): The Lira auth_token.
+        notification (dict): A dict of notification content.
+
+    Returns:
+        requests.Response: The response object returned by Lira.
+    """
+    response = requests.post(url=lira_url + 'notifications', json=notification, params={'auth': auth_token})
+    return response
+
+
+def prepare_notification(bundle_uuid, bundle_version, subscription_id, label=None, transaction_id=None):
+    """Compose the notification content from given values.
+
+    Args:
+        bundle_uuid (str): A valid Blue Box bundle uuid.
```

Comment:
I think 'valid' is implied for these parameters, so you could consider taking that out